.cirrus.yml

```diff
@@ -122,8 +122,8 @@ gradle_qa_task:
       - GRADLE_VERSION: 8.3
       - GRADLE_VERSION: 8.0
       - GRADLE_VERSION: 7.6.4
-#- GRADLE_VERSION: 7.5.1
-#        ANDROID_GRADLE_VERSION: 7.1.0
+      - GRADLE_VERSION: 7.5.1
+        ANDROID_GRADLE_VERSION: 7.1.0
```

Comment:
Consider re-enabling that configuration as part of a separate commit

Reply:
I think it is a good idea, I thought about splitting but was a bit too lazy. :+1: